security_best_practices/security_best_practices.sp

```diff
@@ -0,0 +1,243 @@
+locals {
+  tailscale_common_tags = merge(local.tailscale_compliance__common_tags, {
+    tailscale = "true"
+  })
+}
+
+variable "api_key_id" {
```

Comment:
@madhushreeray30 What types of keys are we asking users to set in this variable? And how dangerous is it if the key IDs were ever exposed? Are they like AWS access key IDs, i.e., not as dangerous if someone only knows the access key ID since there's a secret key, or is the API key ID enough to gain access as that user or perform other unauthorized actions?